server/clip_server/model/clip.py

```diff
@@ -50,12 +51,20 @@
 }
 
 
-def _download(url: str, root: str, with_resume: bool = True):
+def _download(
+    url: str, md5: str, root: str, with_resume: bool = True, max_attempts: int = 3
```

Comment:
`md5` should also be optional. If md5 is not provided, then the checking step is ignored